src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "acl_pooling.hpp"
+#include "acl_utils.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace arm_compute;
+
+AclPoolingExecutor::AclPoolingExecutor(const ExecutorContext::CPtr context) : PoolingExecutor(context) {}
+
+bool AclPoolingExecutor::init(const PoolingAttrs& poolingAttrs,
+                             const std::vector<MemoryDescPtr>& srcDescs,
+                             const std::vector<MemoryDescPtr>& dstDescs,
+                             const dnnl::primitive_attr &attr) {
+    auto srcDims = srcDescs[0]->getShape().getStaticDims();
+    auto dstDims = dstDescs[0]->getShape().getStaticDims();
+
+    TensorInfo srcTensorInfo = TensorInfo(shapeCast(srcDims), 1,
+    precisionToAclDataType(srcDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(srcDescs[0]));
+    TensorInfo dstTensorInfo = TensorInfo(shapeCast(dstDims), 1,
+    precisionToAclDataType(dstDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(dstDescs[0]));
+
+    unsigned int pad_left = (poolingAttrs.data_pad_begin.size() == 2) ? poolingAttrs.data_pad_begin[1] : 0;
+    unsigned int pad_right = (poolingAttrs.data_pad_end.size() == 2) ? poolingAttrs.data_pad_end[1] : 0;
+    unsigned int pad_top = poolingAttrs.data_pad_begin[0];
+    unsigned int pad_bottom = poolingAttrs.data_pad_end[0];
+    unsigned int kernel_w = (poolingAttrs.kernel.size() == 2) ? poolingAttrs.kernel[1] : poolingAttrs.kernel[0];
```

Comment:
As I can see ACL support Pooling3dLayer. Can we use to cover 5D tensors as well?

Reply:
I added NEPooling3dLayer to ACL executor, however there is an issue to use it.
NEPooling3dLayer supports NDHWC only and useACL flag is initialised in getSupportedDescriptors().
So, we need to define the layout in getSupportedDescriptors() - I believe there is no way to do that. Or am I wrong?

Reply:
Yes. Need new functionality on Executors level (configuration reporting API). Lets postpone.